### PR TITLE
Remove lingering serf bin

### PR DESCRIPTION
**Pull request by bernardjkim**
## Description
Remove lingering serf bin/refs

## Linked tickets and PRs
- Requires https://github.com/gravitational/satellite/pull/295

## TODO
- [x] Update upstream references / tags / versions after upstream PR merges (linked above)

### CODE REVIEW

File: build.assets/makefiles/base/agent/agent.mk
Comment (wadells) [inline build.assets/makefiles/base/agent/agent.mk:3]: `$(BINARIES)` used to depend on `planet-agent.service` is that no longer needed?

**Review by wadells — APPROVED** [review]

**Review by knisbet — APPROVED** [review]
it's always great when things get smaller by 30k lines! LGTM once walt's not has been looked at.

File: build.assets/makefiles/base/agent/agent.mk
Comment (bernardjkim) [reply build.assets/makefiles/base/agent/agent.mk:3]: Good catch. This target should still depend on `planet-agent.service`.

**Pull request merged**
